.github/workflows/git-builds.yaml

```diff
@@ -11,7 +11,20 @@ env:
   ProjectFile: "src/NexusMods.App/NexusMods.App.csproj"
 
 jobs:
+  transformInputs:
+    runs-on: ubuntu-latest
+    outputs:
+      ProjectFile: ${{ steps.setOutputs.outputs.projectFile }}
+
+    steps:
+      - name: Set Outputs
+        id: setOutputs
+        shell: pwsh
+        run: |
+          echo "projectFile=${{ env.ProjectFile }}" >> $env:GITHUB_OUTPUT
```

Comment:
I'm a bit confused, why is it using `pwsh` on ubutu-latest?

Reply:
I just prefer this syntax over bash. It makes it clearer that we're writing to an environment variable.